runisort

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 set -e
-isort --check-only --recursive --diff --skip migrations || exit 1
+isort --check-only --recursive --diff || exit 1
+
```

Comment:
There is an extra space here. Last line needs to be an empty line. :smile:

Reply:
Can you suggest me what should I write instead of updated and sorry about this.

Reply:
Can I change it because I have squashed it?

Reply:
ok done i changed commit messages relevantly and removed extra line

Reply:
i did. Yeah you are write i had not think about it that way now i will keep this in mind. Thanks :)

Reply:
ok did change the body. I thought i have to put fixes issue number.